gravitee-apim-portal-webui/src/app/pages/application/application-creation/application-creation.component.ts

```diff
@@ -71,8 +71,8 @@ type ApplicationFormType = FormGroup<{
 
 function mapToApplicationInput(rawValue): ApplicationInput {
   const result = rawValue as ApplicationInput;
-  if (rawValue.oauth !== undefined) {
-    result.settings.oauth.additional_client_metadata = rawValue.oauth.additionalClientMetadata.reduce((acc, { key, value }) => {
+  if (rawValue.settings.oauth !== undefined) {
```

Comment:
```suggestion
  if (rawValue.settings.oauth) {
```
This means that if the value is truthy (not null or undefined or false). This is more bug-proof for this use case because if the value is `null`, it will not be evaluated.